DESCRIPTION

```diff
@@ -38,7 +38,7 @@ Authors@R:
              role = "ctb",
              email = "mutlusun@users.noreply.github.com",
              comment = c(ORCID = "0000-0002-6021-804X")))
-Maintainer: Dominique Makowski <dom.makowski@gmail.com>
+Maintainer: Rémi Thériault <remi.theriault@mail.mcgill.ca>
```

Comment:
In addition to this, you will also need to change your role from `"aut"` to `c("aut", "cre")`.

Reply:
And remove `"cre"` from Dom's role. Maintainers are also expected to be the creators.

Reply:
Weird, they should make another role like `mtn` or something 😤